README.md

```diff
@@ -1,2 +1,100 @@
-# example-grpc-server
-A simple Go grpc server app for Deis, the open source PaaS https://deis.com
+# Go GRPC Server Quick Start Guide
+
+This guide will walk you through deploying a Go [grpc][grpc] server on [Deis Workflow][]. The server is made non-routable outside of the cluster and a Go [grpc client][grpcclient] need to be installed to talk with the server. The server runs on port 5000 by default and can be configured using `deis config:set PORT=<>`.
```

Comment:
The "port 5000" link probably isn't necessary, since that's a given with all of our example apps.